### PR TITLE
add new utility functions

### PR DESCRIPTION
**Pull request by dbasunag**
Utility functions to be used in test repository

### CODE REVIEW

**Comment by redhat-qe-bot1** [comment]
Report bugs in [Issues](https://github.com/myakove/github-webhook-server/issues)

The following are automatically added:
 * Add reviewers from OWNER file (in the root of the repository) under reviewers section.
 * Set PR size label.
 * New issue is created for the PR. (Closed when PR is merged/closed)
 * Run [pre-commit](https://pre-commit.ci/) if `.pre-commit-config.yaml` exists in the repo.

Available user actions:
 * To mark PR as WIP comment `/wip` to the PR, To remove it from the PR comment `/wip cancel` to the PR.
 * To block merging of PR comment `/hold`, To un-block merging of PR comment `/hold cancel`.
 * To mark PR as verified comment `/verified` to the PR, to un-verify comment `/verified cancel` to the PR.
        verified label removed on each new commit push.
 * To cherry pick a merged PR comment `/cherry-pick <target branch to cherry-pick to>` in the PR.
    * Multiple target branches can be cherry-picked, separated by spaces. (`/cherry-pick branch1 branch2`)
    * Cherry-pick will be started when PR is merged
 * To build and push container image command `/build-and-push-container` in the PR (tag will be the PR number).
 * To add a label by comment use `/<label name>`, to remove, use `/<label name> cancel`
 * To assign reviewers based on OWNERS file use `/assign-reviewers`
 * To check if PR can be merged use `/check-can-merge`

<details>
<summary>Supported /retest check runs</summary>

 * `/retest tox`: Retest tox
 * `/retest python-module-install`: Retest python-module-install

</details>

<details>
<summary>Supported labels</summary>

 * hold
 * verified
 * wip
 * lgtm

</details>

**Review by jpeimer — APPROVED** [review]

**Comment by myakove** [comment]
Change the PR title to have a meaningful description
Add body that describe what was added in this PR.

**Review by jpeimer — CHANGES_REQUESTED** [review]
Please add a more meaningful title and description

**Review by jpeimer — DISMISSED** [review]

File: operator_utilities/infra.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
+from simple_logger.logger import get_logger
```
Comment (myakove) [inline operator_utilities/infra.py:1]: all functions in this file are not operators related.

File: operator_utilities/operator.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    """
+    csv = ClusterServiceVersion(name=csv_name, namespace=namespace)
+    if csv.exists:
+        return csv
+    raise ResourceNotFoundError(f"CSV: {csv_name} not found in namespace: {namespace}")
+
+
+def get_subscription(subscription_name, namespace):
+    """
+    gets subscription by name
```
Comment (myakove) [inline operator_utilities/operator.py:29]: not operators related functions

File: operator_utilities/operator.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+        namespace=namespace,
+        subscription_name=subscription_name,
+    )
+    return get_csv(
+        csv_name=cnv_subscription.instance.status.installedCSV,
+        namespace=namespace,
+    )
+
+
+def get_catalog_source(catalogsource_name, catalogsource_namespace):
```
Comment (myakove) [inline operator_utilities/operator.py:74]: not operators related

File: operator_utilities/hyperconverged.py [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+from ocp_resources.resource import Resource
+from operator_utilities.exceptions import (
+    HyperconvergedNotHealthyCondition,
+    HyperconvergedSystemHealthException,
+)
+
+LOGGER = get_logger(name=__name__)
+
+
+def get_hyperconverged_resource(hyperconverged_name, namespace_name):
```
Comment (myakove) [inline operator_utilities/hyperconverged.py:16]: Please start using typing for all

**Review by myakove — CHANGES_REQUESTED** [review]
Please add tests, this repo does not have code (beside 1 function).

File: operator_utilities/operator.py (hunk 3 above)
Comment (dbasunag) [reply operator_utilities/operator.py:74]: Catalogsource is part of OLM. https://olm.operatorframework.io/docs/concepts/crds/catalogsource/

File: operator_utilities/operator.py (hunk 2 above)
Comment (dbasunag) [reply operator_utilities/operator.py:29]: Ditto, subscription is part of OLM

File: operator_utilities/hyperconverged.py (hunk 4 above)
Comment (dbasunag) [reply operator_utilities/hyperconverged.py:16]: Ok, I will do this.

File: operator_utilities/infra.py (hunk 1 above)
Comment (dbasunag) [reply operator_utilities/infra.py:1]: Renaming file to olm.py

**Comment by dbasunag** [comment]
> Please add tests, this repo does not have code (beside 1 function).

Will do. In a follow up PR.

**Pull request merged**